gradle-plugin/build.gradle

```diff
@@ -62,9 +60,9 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
   kotlinOptions {
     jvmTarget = JavaVersion.VERSION_1_8
 
-    // Because Gradle hardcodes Kotlin versions we have warnings during compilation.
-    allWarningsAsErrors = rootProject.ext.ci &&
-        VersionNumber.parse(rootProject.ext.kotlinVersion).baseVersion < VersionNumber.parse("1.5.0").baseVersion
+    // Because Gradle hardcodes Kotlin versions we have warnings during compilation. We might need
+    // disable this setting when upgrading to 1.5.0 stable.
```

Comment:
nit: missing the word "to": "We might need to disable this setting..."

Reply:
Looking at it again, I wonder if this whole comment should be moved to dependencies.gradle, since it's referencing the version-based logic that you moved there.

Reply:
Uff, I saw your comment after merging. But I don't want to move the comment either, because it only applies to the gradle plugin (this module) and nowhere else.